opencadd/structure/superposition/engines/base.py

```diff
@@ -32,3 +32,6 @@ def calculate(self, structures, *args, **kwargs):
 
     def _calculate(self, structures, *args, **kwargs):
         raise NotImplementedError("Reimplement in your subclass")
+
+    def _safety_checks(self):
+        raise NotImplementedError("Reimplement in your subclass")
```

Comment:
Should it also be implemented for mda for consitency?
The method would just pass for mda.